src/handler/http/request/dashboards/mod.rs

```diff
@@ -208,7 +229,16 @@
     dashboards::move_dashboard(&org_id, &dashboard_id, &folder.from, &folder.to).await
 }
 
+fn get_query_params(req: HttpRequest) -> Query<HashMap<String, String>> {
+    web::Query::<HashMap<String, String>>::from_query(req.query_string()).unwrap()
+}
```

Comment:
In this case I just moved this code into it's own function, but I do agree with the recommendation. It would involve making some changes to other functions as well to handle the errors that get bubbled up but I'm happy to make these changes.